src/parser/jobs/ast/modules/ArcanaGroups.ts

```diff
@@ -1,55 +1,55 @@
-import ACTIONS from 'data/ACTIONS'
-import STATUSES from 'data/STATUSES'
+import {ActionRoot} from 'data/ACTIONS/root'
+import {StatusRoot} from 'data/STATUSES/root'
 // Grouped cards for future convenience - whatever processing desired
 
 /* Grouped actions */
 
-export const PLAY = [
```

Comment:
This change breaks the grouping of card actions on the timeline, since Cooldowns.tsx is importing PLAY for the list of actions to merge under the "Play" heading, and Core Cooldowns expects a list of Action ID numbers, not action name strings.

Reply:
Oh...
I'll feeex